pkg/kubelet/prober/worker.go

```diff
@@ -230,7 +238,7 @@ func (w *worker) doProbe() (keepGoing bool) {
 	if c.Started != nil && *c.Started {
 		// Stop probing for startup once container has started.
 		if w.probeType == startup {
-			return true
+			return false
 		}
 	} else {
 		// Disable other probes until container has started.
```

Comment:
is this comment correct?

Reply:
Yes, we skip the probing part, but still leave the worker running.